src/components/Bucketlist.js

```diff
@@ -34,26 +55,53 @@ class Bucketlist extends Component {
             headers: {"Authorization": localStorage.getItem('token')}
         }).then((response) => {
             this.setState({
-                bucketlists: response.data.bucketlist
+                bucketlists: response.data.bucketlist,
+                next_page :response.data.next_page,
+                prev_page :response.data.prev_page,
             })
-        }).catch((error)=>{
-            console.log(error)
-            
-        })
+        }).catch((error)=>{})
```

Comment:
Do consider doing something with your .catch sections. To avoid having empty blocks such as these. Cool?